deep-learning/_posts/2017-08-04-A-Fast-RCNN.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: review
+title:  "A-Fast-RCNN: Hard Positive Generation via Adversary for Object Detection"
+tags:   deep-learning localization CNN GAN classification detection
+author: Clément Zotti
+pdf:   "https://arxiv.org/pdf/1704.03414.pdf"
+
+cite:
+  authors: "X. Wang, A. Shrivastava, A. Gupta"
+  title:   "A-Fast-RCNN: Hard Positive Generation via Adversary for Object Detection"
+  venue:   "CVPR 2017"
+---
+
+Paper :
+   - [code](https://github.com/xiaolonw/adversarial-frcnn)
+
+This paper shows a way to train an adversarial network to generate occlusion/deformation changes to help an object detector network to generalize better.
+
+As an object detector network they use [Faster-RCNN]({% post_url /deep-learning/2017-05-14-fastrcnn %}).
+
+## Model
+
+The adversarial model has two parts (Fig.). The first one is a spatial dropout occlusion (ASDN), it generates occlusion maps on the object features after the ROI layer.
+To select which pixel to mask, they use importance sampling on the mask and keep only the top $$ \frac{1}{3} $$ pixels.
+
+The second part is a spatial transformer network (ASTN), it generates deformation on the object features. The deformation is mostly focused on the rotation. They found that limiting the rotation is a key factor in their experiment to not have upside down rotations and fool the prediction network. The rotation limit is in between $$ -10^\circ $$ to $$ 10^\circ $$. The feature maps are divided into 4 blocks given the channel axis and different rotations are applied on these blocks.
```

Comment:
> divided into 4 blocks given the channel axis

I would replace "given" by "along"

This clarifies things! During your presentation I thought it was divided spatially!

Reply:
Yes, I spell it wrong during the presentation.